src/ldc/eh/win32.d

```diff
@@ -155,56 +155,16 @@ CatchableType* getCatchableType(TypeInfo_Class ti)
     if (auto p = ti in catchableHashtab)
         return p;
 
-    static size_t mangledNameLength(string s)
-    {
-        size_t len = 2 + s.length; // "_D" + identifier + 1 digit per dot
-        for (size_t q = 0; q < s.length; )
-        {
-            size_t p = q;
-            while (p < s.length && s.ptr[p] != '.')
-                p++;
-            for (size_t r = p - q; r >= 10; r /= 10) // add digits for length >= 10
-                len++;
-            q = p + 1;
-        }
-        return len;
-    }
-
-    static void mangleName(string s, char* buf)
-    {
-        *buf++ = '_';
-        *buf++ = 'D';
-        for (size_t q = 0; q < s.length; )
-        {
-            size_t p = q;
-            while (p < s.length && s.ptr[p] != '.')
-                p++;
-            size_t digits = 10;
-            size_t len = p - q;
-            for ( ; len >= digits; digits *= 10) {}
-            for (digits /= 10; digits > 1; digits /= 10)
-            {
-                size_t dig = len / digits;
-                *buf++ = cast(char)('0' + dig);
-                len -= dig * digits;
-            }
-            *buf++ = cast(char)('0' + len);
-            memcpy(buf, s.ptr + q, p - q);
-            buf += p - q;
-            q = p + 1;
-        }
-        *buf = 0;
-    }
-
-    size_t mangledLength = mangledNameLength(ti.name) + 1;
-    size_t sz = TypeDescriptor!1.sizeof + mangledLength;
+    size_t sz = TypeDescriptor!1.sizeof + ti.name.length + 2;

```

Comment:
`+ 2` is obsolete here I think - `TypeDescriptor!1.name` already provides space for 2 chars (`D` prefix and `\0` suffix`).